src/cs3panel.tsx

```diff
@@ -89,18 +91,61 @@ export class Cs3HeaderWidget extends ReactWidget {
     }
 }
 
+export const Bottom = (props: BottomProps): JSX.Element => {
+
+    const [text, setText] = useState(props.message);
+
+    const setLabel = async () => {
+        const showHidden: boolean = await props.db.fetch('showHidden') as boolean;
+        if (showHidden == undefined || !showHidden) {
+            const hiddenFilesNo: number = await props.db.fetch('hiddenFilesNo') as number;
+            setText('Hidden files ' + hiddenFilesNo + ' (show)')
+        } else {
+            setText('Cover hidden files')
+        }
+    }
+
+    props.browser.model.pathChanged.connect( async (browser, args) => {
+        await setLabel()
+    })
+
+    props.browser.model.refreshed.connect(async (browser) => {
+        await setLabel()
+    })
+
+    return (
+        <div className={'jp-bottom-div'} onClick={async () => {
```

Comment:
Do we even change `showHidden` to true or false (I'm only seeing fetch)? SHouldn't we be doing it here?